vsts/scripts/copySdksFromProdToStorageAccount.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -ex
+
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && pwd )
+source $REPO_DIR/platforms/__common.sh
+
+azCopyDir="/tmp/azcopy-tool"
+
+function blobContainerExistsInDestination() {
+	local containerName="$1"
+	local exitCode=1
+	curl -I "$destinationSdk/$containerName?restype=container" 2> /tmp/curlError.txt 1> /tmp/curlOut.txt
+	grep "HTTP/1.1 200 OK" /tmp/curlOut.txt &> /dev/null
+	exitCode=$?
+	rm -f /tmp/curlOut.txt
+	rm -f /tmp/curlError.txt
+	if [ $exitCode -eq 0 ]; then
+		return 0
+	else
+        return 1
+	fi
+}
+
+function copyBlobContainerFromProdToDestination() {
+    local platformName="$1"
+
+    if shouldOverwriteSdk || shouldOverwritePlatformSdk $platformName; then
+        echo
+        echo "Overwriting blob container '$platformName' in storage account '$destinationSdk'."
+        # azcopy copy [source] [destination] [flags]
+        "$azCopyDir/azcopy" copy \
+            "$PROD_SDK_STORAGE_BASE_URL/$platformName/$PROD_STORAGE_SAS_TOKEN" \
```

Comment:
Even though you have it documented a couple of lines above, it would personally make me feel more comfortable if we used the explicit parameter name when providing these two arguments, just so there's no ambiguity as to what argument is associated with what parameter, and in the (hopefully) very very very slim chance a breaking change was made that flipped positional arguments, we are shielded from it.

Reply:
hmmm so I'm actually not sure if there is a way to do this? I've been looking around at the docs and such and can't find "source" or "destination" under any flags that I can pass in. did some testing locally and still couldn't determine the correct way to do it:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/107068277/182869566-8fadf685-1754-45a2-9b8c-fb6900ecb517.png">